checker/tests/nullness/RequiresNonNullTest.java

```diff
@@ -143,4 +143,10 @@ void notHiddenTest() {
     void test() {
         o = null;
     }
+
+    @RequiresNonNull("thisShouldIssue1Error")
+    // Test case for Issue 1051
+    // https://github.com/typetools/checker-framework/issues/1051
+    //:: error: (flowexpr.parse.error)
```

Comment:
It's fine if you want to create one.  Note, that the `flowexpr.parse.error` includes the expression string that failed to parse.  So the user can figure out which annotation failed, though it would be nicer if the error were on the correct annotation.